BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/PerformanceConfiguration.cs

```diff
@@ -37,6 +38,24 @@ public class PerformanceConfiguration
 
         public String[] TracePropagationUrlMatchPatterns;
 
+
+        public void AddOnSpanEndCallback(Func<Span, bool> callback)

```

Comment:
This matches the notifier:

```suggestion
        public void AddOnSpanEnd(Func<Span, bool> callback)

```